demo/lib/TransformationControls/addHideRedlinksControl.js

```diff
@@ -1,12 +1,18 @@
 /* eslint-disable require-jsdoc */
 
+const hideRedLinks = (iframeWindow, iframeDocument) => {
+  event.target.disabled = true
+  if (iframeWindow.pagelib.c1) {
+    // TODO: hideRedLinks is not exposed in the abstraction layer
```

Comment:
We're not going to add this AFAIK. The service side already does this, so there is no reason to expose it.